### PR TITLE
:bug: Source Google Search Console: migrate to advanced auth

### PR DESCRIPTION
**Pull request by davydov-d**
## What
#26233 

## How
Remove `authSpecification` from connector spec in favour of `advancedAuth`

_note: this change can not be testes locally due to redirect_uri mismatch_

### CODE REVIEW

**Review by bazarnov — APPROVED** [review]

File: airbyte-integrations/connectors/source-google-search-console/source_google_search_console/spec.json [hunk 1]
```diff
@@ -126,12 +126,48 @@
       }
     }
   },
-  "authSpecification": {
-    "auth_type": "oauth2.0",
-    "oauth2Specification": {
-      "rootObject": ["authorization", 0],
-      "oauthFlowInitParameters": [["client_id"], ["client_secret"]],
-      "oauthFlowOutputParameters": [["access_token"], ["refresh_token"]]
+  "advanced_auth": {
```
Comment (girarda) [inline airbyte-integrations/connectors/source-google-search-console/source_google_search_console/spec.json:129]: do we automatically convert the config object behind the scene or is this a breaking change?

Comment (davydov-d) [reply airbyte-integrations/connectors/source-google-search-console/source_google_search_console/spec.json:129]: @girarda this is neither a breaking change, nor the config object has to be changed. This is just another way to tell the server how to treat the config while going through the oauth flow

**Review by girarda — APPROVED** [review]

**Review by lazebnyi — APPROVED** [review]

**Comment by davydov-d** [comment]
Blocked because we dont have valid UI credentials for this source so can not even set up a connection although authentication succeeds
image

**Pull request merged**